graylog2-web-interface/src/components/common/Select.tsx

```diff
@@ -473,7 +473,10 @@ class Select extends React.Component<Props, State> {
     if (value && allowCreate) {
       formattedValue = this._formatInputValue(value);
     } else {
-      formattedValue = (value || '').split(delimiter).map((v) => options.find((option) => option[valueKey || ''] === v));
+      formattedValue = (value instanceof String
```

Comment:
E.g. the stream select does not display the selected options when I select multiple streams. It looks like the component receives the value as a string literal which is not an instance of `String`. It works with `typeof value === 'string'`.